### PR TITLE
feat(crux-ui): breadcrumb home icon

### PR DESCRIPTION
**Pull request by robot9706**
Changed the breadcrumb home icon to reflect which sidebar menu it goes back to.

### CODE REVIEW

File: web/crux-ui/src/components/main/sidebar.tsx [hunk 1]
```diff
@@ -21,85 +21,99 @@ import Link from 'next/link'
...
+  {
+    title: 'components',
+    items: [
+      {
+        icon: '/servers.svg',
+        text: 'nodes',
+        link: ROUTE_NODES,
+      },
+      {
+        icon: '/copy.svg',
```
Comment (polaroi8d) [inline web/crux-ui/src/components/main/sidebar.tsx:58]: Don't we want to rename this and `server.svg` to the exact menu? like `node.svg` and `registries.svg`?

**Review by polaroi8d — COMMENTED** [review]
Minor issue / question added.

File: web/crux-ui/src/components/main/sidebar.tsx (hunk 1 above)
Comment (robot9706) [reply web/crux-ui/src/components/main/sidebar.tsx:58]: `copy.svg` is used elsewhere, so we could keep 2 copies of it one for the sidebar (`registries.svg`) and one for everything else (`copy.svg`).

Comment (polaroi8d) [reply web/crux-ui/src/components/main/sidebar.tsx:58]: Ahh not the right implementation, right now its okey. Approved.

**Review by polaroi8d — APPROVED** [review]
LGTM

**Pull request merged**